libsweep/src/sweep.cc

```diff
@@ -19,25 +22,17 @@ typedef struct sweep_device {
 
 #define SWEEP_MAX_SAMPLES 4096
 
-typedef struct sweep_scan {
+struct sweep_scan {
```

Comment:
Must have slipped through, will fix. 
I originally had all the typedefs removed throughout the library but wanted to keep this PR clean of it (there is already going on quite a lot anyway), so I reverted those changes - except that I apparently overlooked this one.